### ISSUE TITLE
feat: support the report for `secret` scan

### ISSUE THREAD

**Issue by afdesk** [afdesk/scan2html#25]
We should support the reports for `secret` scan.

there is a simple `Dockerfile`
```Dockerfile
FROM nginx:alpine
ENV AWS_SECRET_ACCESS_KEY="1111111111111/2222222/333333333333333333"
CMD ["nginx", "-g", "daemon off;"]
```
Trivy detects a secret here:
```sh
$ trivy fs .
2023-03-10T17:22:44.410+0600	INFO	Vulnerability scanning is enabled
2023-03-10T17:22:44.410+0600	INFO	Secret scanning is enabled
2023-03-10T17:22:44.410+0600	INFO	If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2023-03-10T17:22:44.410+0600	INFO	Please see also https://aquasecurity.github.io/trivy/v0.38/docs/secret/scanning/#recommendation for faster secret detection
2023-03-10T17:22:44.411+0600	INFO	Number of language-specific files: 0

Dockerfile (secrets)

Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 1)

CRITICAL: AWS (aws-secret-access-key)
═══════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════
AWS Secret Access Key
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
 Dockerfile:3
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   FROM nginx:alpine
   2   
   3 [ ENV AWS_SECRET_ACCESS_KEY="****************************************"
   4   
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
```

Now the result is next:
```sh
$ trivy scan2html fs . r.html
```

![image](https://user-images.githubusercontent.com/19297627/224304552-1661edf2-f228-408d-a143-0b6419dda991.png)

**Comment by nikpivkin** [comment]
@malikovmm Is it in the process or can I take it?

**Comment by nikpivkin** [comment]
@afdesk can you assign it to me? I'm in progress

**Comment by afdesk** [comment]
> @afdesk can you assign it to me? I'm in progress

I'd like that Maksim figures out this issue. because it needs to change the layout.

@nikpivkin could you take a look at the GitLab integration?

**Comment by nikpivkin** [comment]
@afdesk ok, of course

**Comment by afdesk** [comment]
closed via #33
